### PR TITLE
Update my-events and refactor javascript

### PR DESCRIPTION
**Pull request by fang-helen**
Added a function that fetches created and saved events from the servlet on loading my-events.html.

Also did some refactoring on script.js -- grouped functions based on which page(s) they apply to.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -433,7 +253,33 @@ async function getEvents(url, index, option) {
 
   var eventListElement = eventListElements[index];
   eventListElement.innerHTML = '';
+
+  if(events.length == 0) {
```
Comment (0tpyrk) [inline src/main/webapp/script.js:257]: Spaces! :)

Comment (fang-helen) [reply src/main/webapp/script.js]: oh no

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -179,261 +201,103 @@ function generateMobileNavLayout() {
...
+      'sit amet mattis libero. Sed tincidunt arcu in justo...',
+  date: 'Sunday, June 21, 2020',
+  startTime: '1:00 PM',
+  distance: '6 miles away',
+  streetAddress: 'Main St',
+  city: 'Los Angeles',
+  state: 'CA',
+  attendeeCount: 12,
+  tags: ['education'],
+  url:'/display-event.html'};
```
Comment (tc466) [inline src/main/webapp/script.js:242]: space after 'url'

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -13,31 +13,47 @@
...
-/* determines which stylesheet to use and generates nav bar*/
-function loadActions() {
+/**
+ * determines which stylesheet to use and generates nav bar
+ *
+ * @param {function} doAfter used to specify actions during checkLogin()
+ */
+function loadActions(doAfter) {
   const styleSheetElement = document.getElementById('style');
   if (window.innerWidth >= window.innerHeight) {
```
Comment (tc466) [inline src/main/webapp/script.js:29]: I will update the eslint config to include pre-defined globals such as 'window'. For now, ignore this lint error.

Comment (fang-helen) [reply src/main/webapp/script.js:29]: ok, thank you!

File: src/main/webapp/script.js (hunk 2 above)
Comment (fang-helen) [reply src/main/webapp/script.js]: fixed!

**Pull request merged**